c/CHANGELOG.rst

```diff
@@ -6,6 +6,13 @@ In development.
 
 **Breaking changes**
 
+- Mutations now have a double-precision floating-point ``time`` column. This is
+  optional in the file format if it is missing ``NAN`` is used for all mutations.
```

Comment:
```suggestion
- Mutations now have an optional double-precision floating-point ``time`` column. If not specified, this defaults to a particular NaN value indicating that the time is unknown.
```

Reply:
Also, I guess this isn't a breaking change any more - move to new features?

Reply:
Still the breaking change to `add_row` but the rest is moved in https://github.com/tskit-dev/tskit/pull/672/commits/676d83276d06a979f2a3f5e840c974928e193fd4